doc/source/how-to/documenting.rst

```diff
@@ -506,49 +505,266 @@ Your next step will be to upload the documentation artifact. Assuming your
 artifacts as a convenient zip and to open the documentation by opening
 ``index.html``.
 
+
+Deploying to GitHub Pages
++++++++++++++++++++++++++
 Next, deploy your documentation to the ``gh-pages`` branch via using the
-``JamesIves/github-pages-deploy-action`` action:
+`JamesIves/github-pages-deploy-action
+<https://github.com/JamesIves/github-pages-deploy-action>`_ action.
 
-.. code-block:: yaml
+.. admonition:: Deploying to another repository.
+
+   If you are planning to deploy documentation to another repository rather than
+   the one for your project, make sure you create this new repo before deploying
+   for the first time your documentation.
 
-    - name: Deploy
-      if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
-      uses: JamesIves/github-pages-deploy-action@4.3.0
-      with:
-        branch: gh-pages
-        folder: doc/build/html
-        clean: true
+The following job step shows the logic for deploying. If you wish to deploy to
+another repository, make sure to uncomment the ``repository-name`` line and
+declare the name of your documentation repository:
+
+.. tabs::
+
+    .. group-tab:: Using ``tox``
+
+        .. code-block:: yaml
+        
+            - name: Deploy
+              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+              uses: JamesIves/github-pages-deploy-action@4.3.0
+              with:
+                token: ${{ secrets.GITHUB_TOKEN }}
+                # repository-name: pyansys/repository-name
+                branch: gh-pages
+                folder: .tox/doc_out
+                clean: true
+
+    .. group-tab:: Without Using ``tox``
+
+        .. code-block:: yaml
+
+            - name: Deploy
+              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+              uses: JamesIves/github-pages-deploy-action@4.3.0
+              with:
+                token: ${{ secrets.GITHUB_TOKEN }}
+                # repository-name: pyansys/repository-name
+                branch: gh-pages
+                folder: doc/_build/html
+                clean: true
+
+
+Notice that for previous job steps, a ``GITHUB_TOKEN`` is required. GitHub
+automatically generates the token ``GITHUB_TOKEN`` which you can use as
+``token`` for deploying documentation to the same repo. However, if you are
+planning to deploy to another repo this token does not have the permissions to
+do it.
+
+In that case, there are two options for documentation deployment, using a bot
+and using a personal access token (PAT). Depending on your profile (how many
+organizations you work in, the rights in the different repositories, etc.), using
+a PAT can be potentially dangerous because PAT are not restricted to defined
+repositories, rather have general permissions. This means that a PAT with
+`repository-write` permission can write in any repo in any organization that the
+PAT creator can access.
```

Comment:
```suggestion
In this case, there are two options for documentation deployment, using a bot
and using a personal access token (PAT). Depending on your profile (such as how many
organizations you work in and your permissions for different repositories), using
a PAT can be potentially dangerous because a PAT is not restricted to defined
repositories but rather has general permissions. This means that a PAT with
`repository-write` permission can write in any repository in any organization that the
PAT creator can access.
```